stl/inc/chrono

```diff
@@ -2426,7 +2553,10 @@ namespace chrono {
             _Unique_lock _Lk(_Tzdb_mutex);
             auto [_Leap_sec, _All_ls_positive] = _Xtzdb_generate_leap_seconds(_Tzdb_list.front().leap_seconds.size());
             if (!_Leap_sec.empty()) {
-                _Tzdb_list.emplace_front(tzdb{_STD move(_Leap_sec), _All_ls_positive});
+                decltype(tzdb::time_zones) _Time_zones{_Tzdb_list.front().time_zones};
+                decltype(tzdb::links) _Links{_Tzdb_list.front().links};

```

Comment:
I think these could be `auto`.

Reply:
Can't use copy constructors anyway because the elements are not copyable (after updating ctors). Now using `std::transform`.